src/views/dissolved-search/index.html

```diff
@@ -44,21 +44,21 @@ <h1 class="govuk-heading-xl">
         }) }}
         {{ govukRadios({
             classes: "govuk-radios--inline",
-            idPrefix: "which-company-name-to-search",
-            name: "changed-name",
+            idPrefix: "changed-name",
+            name: "changedName",
             fieldset: {
               legend: {
                 text: "Which company names do you want to search?"
               }
             },
             items: [
               {
-                value: "name-at-dissolution",
+                value: "bestMatch",
```

Comment:
This value should be nameAtDissolution, although we do not use this value for logic right now it still displays in the url